.travis.yml

```diff
@@ -2,6 +2,7 @@ language: php
 php:
   - 7.2
```

Comment:
please add 7.3

Reply:
ok, but Travis is not configured to run on your (easywelfare) repository...